metagraph/src/cli/config/config.cpp

```diff
@@ -1170,6 +1178,12 @@ void Config::print_usage(const std::string &prog_name, IdentityType identity) {
             fprintf(stderr, "\t-p --parallel [INT] \tmaximum number of parallel connections [1]\n");
             // fprintf(stderr, "\t   --cache-size [INT] \tnumber of uncompressed rows to store in the cache [0]\n");
         } break;
+        case TRANSFORM_ANNO_TAX: {
+            fprintf(stderr, "Usage: %s transform_anno_tax --taxonomic-tree <nodes.dmp> --lookup-table <*.accession2taxid> -o <TAXONOMIC_DB-basename> ANNOT1 [[ANNOT2] ...] [options]\n\n", prog_name.c_str());
```

Comment:
Break the line. It's hard to read here and it will be hard to read in the Usage message in the command line output.